src/svg-renderer.js

```diff
@@ -279,11 +235,19 @@ class SvgRenderer {
 
     /**
      * Serialize the active SVG DOM to a string.
+     * @param {?boolean} fontsInjected True if fonts should be included in the SVG as
```

Comment:
Super nit-pick, but I would prefer Boolean arguments to be phrased in terms of what it means when the value is true. In this case, if the value is true it means that this code should inject the fonts -- not that they've already been injected. I would call it something like `shouldInjectFonts` or `fontInjectionRequested` or something like that.

Reply:
No problem, I think that makes sense